build.sbt

```diff
@@ -7,38 +7,96 @@ organization in ThisBuild := "org.typelevel"
 
 // aliases
 
-addCommandAlias("buildJVM", "catsMtlJVM/test")
+addCommandAlias("buildJVM", "testsJVM/test")
 
 addCommandAlias(
   "validateJVM",
   ";scalastyle;sbt:scalafmt::test;scalafmt::test;test:scalafmt::test;buildJVM;mimaReportBinaryIssues;makeMicrosite")
 
-addCommandAlias("validateJS", ";catsMtlJS/compile;testsJS/test;js/test")
+addCommandAlias("validateJS", ";testsJS/compile;testsJS/test")
 
 addCommandAlias("validate", ";clean;validateJS;validateJVM")
 
 addCommandAlias("gitSnapshots",
                 ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\"")
 
-sbtPlugin := true
```

Comment:
This line was what caused the issue. *\*shakes fist\**

Reply:
Damn, this line...

Reply:
```scala
val total_hours_wasted_here = 3
```